packagelist/debian_12_headless.txt

```diff
@@ -37,12 +36,12 @@ apt-utils
 bsdmainutils
 dbus
 fdisk
+parted
```

Comment:
Why is proximity to fdisk in the list more important than alphabetization and diff-ability against other package lists?

Reply:
Should be fixed now.